routes/web.php

```diff
@@ -91,6 +91,7 @@
 
 // CREATE
 Route::get('event/{year?}/{month?}/{day?}/{templateId?}/create', 'ClubEventController@create');
+Route::post('event/{year?}/{month?}/{day?}/{templateId?}/create', 'ClubEventController@create');
```

Comment:
**Important:**
POSTing to a "create" route is not allowed! 
The "create" route only accepts GET requests and delivers the form for creating a new resource. 
If I understand the intent correctly. this should be a "store" route instead, and the function stores a newly created resource in storage.
And if it was an update, then the "update" route should use PUT and update the specified resource in storage.

ref.: https://laravel.com/docs/5.6/controllers#resource-controllers

Reply:
this route is for changing the template
the old behaviour was raising a warning of the browser because you can loose data when leaving it -> kind of exception -> using exceptions for regular behavior is bad practice
this is a kind of redirect with resuage
PUT will be used for soring data -> not wantet -> Wantet: open create view with filled template data